tests/e2e/utils/startPuppe.js

```diff
@@ -82,11 +83,9 @@ function listenToEvents(page) {
 async function getPage(browser) {
   const page = await browser.newPage();
   page.setCacheEnabled(false);
-  // NOTE: activate to see what happens in puppeteer evaluate function
-  // page.on('console', consoleObj => console.log(consoleObj.text()));
   await page.setViewport({
-    width: 1920,
-    height: 1080
+    width: 1680,
+    height: 950
```

Comment:
why was this done?

Reply:
it was too big for my laptop screen to check it in demo mode